### ISSUE TITLE
path resolving problem

### ISSUE THREAD

**Issue by akitsu-sanae** [haxiomic/dts2hx#23]
I found a problem with dts2hx. This might be a bug.

In https://github.com/akitsu-sanae/dts2hx-issue, `@dts2hx-issue/foo` module in `node_modules` tries to import a class under `@dts2hx-issue/hoge` module.  
TypeScript compiler can handle this code, but `dts2hx` failed with following message
```
> Converting module @dts2hx-issue/foo
> Error: Failed to get sourceFile "/path/to/dts2hx-issue/node_modules/@dts2hx-issue/hoge/index.d.ts"
```
But the file `/path/to/dts2hx-issue/node_modules/@dts2hx-issue/hoge/index.d.ts` exists.

**Comment by haxiomic** [comment]
Hey @akitsu-sanae thanks for the super clear repro! I’ll take a look today

**Comment by haxiomic** [comment]
Found the problem – it was over enthusiastic error checking on my part. Fixed in 0.11.10

Make sure to change to regular quotes before running tho (as back-ticks are an error here)
https://github.com/akitsu-sanae/dts2hx-issue/blob/7604e8da28d74dbe7b28788a8434f3c0421149ad/node_modules/%40dts2hx-issue/hoge/index.d.ts#L1

**Comment by akitsu-sanae** [comment]
Thank you!
